torch/onnx/symbolic_helper.py

```diff
@@ -146,11 +170,7 @@ def wrapper(g, *args, **kwargs):
             if len(kwargs) == 1:
                 assert '_outputs' in kwargs
             return fn(g, *args, **kwargs)
-        # In Python 2 functools.wraps chokes on partially applied functions, so we need this as a workaround
```

Comment:
(Given the previous comment..) Is this change ok with Python 2? CI does not cover ONNX export under python 2.

Reply:
According to this, PyTorch dropped support for Python 2 ~1.5 years ago: https://discuss.pytorch.org/t/dropping-python-2-support-on-january-1st-2020/56730

Is there some reason the code in torch/onnx still needs to support Python 2 when the rest of torch does not?

Reply:
i see, thanks for updating this part.